src/wallet/index.ts

```diff
@@ -300,31 +294,22 @@ export const getWeb3Provider = async (): Promise<Web3Provider> => {
 /**
  * Check if a web3 wallet is installed
  *
- * @returns installed web3 provider or null if none
+ * @returns boolean
  * @throws If this is not run in a browser
  */
-export const checkIsWalletInstalled = () => {
+export const isWeb3WalletInstalled = () => {
   if (typeof window === "undefined") {
     throw new Error("Should run through Browser");
   }
-  // Have to check the ethereum binding on the window object to see if it's installed
-  const { ethereum } = window;
-
-  try {
-    // is there a more agnostic way to check? I know otherwallets use isMetaMask too, but perhaps there are better flags
-    const check = ethereum.isMetaMask;
-  } catch (e) {
-    return null;
-  }
 
-  return ethereum;
+  return window?.ethereum?.isMetaMask || false;
```

Comment:
```
return !!window?.ethereum?.isMetaMask;
```


![Screen Shot 2023-02-09 at 09 55 33](https://user-images.githubusercontent.com/380327/217832211-5e1d6535-1201-40c5-a42e-de96bf20d41f.png)

Reply:
more modern way and more concise. like it!